doc/user/script_description.md

```diff
@@ -2,13 +2,16 @@
 
 Ensure the pre-conditions described [here](../README.md).
 
-The method `caom2utils.fits2caom2.augment` uses the concept of a blueprint to capture the description of a CAOM2 Observation as a 
+The method `caom2utils.caom2blueprint.augment` uses the concept of a blueprint to capture the description of a CAOM2 Observation as a 
 mapping of a Telescope Data Model (TDM) to the CAOM2 data model. This describes how to extend that application to customize the mapping for a `COLLECTION`.
 
 `augment` works by creating or augmenting a CAOM2 Observation record, which can then be loaded via the CADC service.
 
-`augment` creates the Observation record using information contained in a FITS file. The python module `fits2caom2`, from the python package `caom2utils`,
-examines the FITS file and uses a blueprint, embodied in an instance of the ObsBlueprint class, to define default values, override values, and mappings to augment the FITS header. The keywords and values in the augmented FITS header are then used to fill in corresponding CAOM2 entities and attributes.
+`augment` creates the Observation record using information contained in a FITS file. The python module `caom2blueprint`, from the python package `caom2utils`,
+examines the FITS or HDF5 file and uses a blueprint, embodied in an instance of the ObsBlueprint class, to define default values, override values, and mappings to augment the file 
+metadata content. The augmented keywords and values are then 
+checked for consistency with `astropy`'s [WCS package](https://docs.astr. 
```

Comment:
Missing end of line

Reply:
Reworded a bit. Hopefully it's more correct now.